src/app/rpmostree-builtin-upgrade.c

```diff
@@ -120,6 +120,8 @@ rpmostree_builtin_upgrade (int             argc,
       GVariantDict dict;
       g_variant_dict_init (&dict, NULL);
       g_variant_dict_insert (&dict, "mode", "s", check_or_preview ? "check" : "auto");
+      /* override default if we're actually on a tty and not e.g. timer triggered */
+      g_variant_dict_insert (&dict, "redirect-output", "b", !glnx_stdout_is_tty ());
```

Comment:
Hmm, what about the other cli options like `rebase`?

Reply:
We default to `FALSE` on the daemon side (i.e. current behaviour). This is just to override to `TRUE` on the client-side so that if someone (most likely a developer) directly calls `upgrade --trigger-automatic-update-policy`, they *do* get output to their terminal.

Actually, hmm, we should be defaulting to `TRUE` on the daemon side for `AutomaticUpdateTrigger`, which we're not doing currently. It just works right now because the client always specifies it, even in the `TRUE` case. Will fix up!